sympy/matrices/tests/test_decompositions.py

```diff
@@ -215,6 +215,31 @@ def test_QR_trivial():
     assert R.is_upper
     assert A == Q*R
 
+
+def _numerically_equal(A, B, tol=1e-15):
```

Comment:
Maybe it's better to use something like `np.allclose` which has both an absolute and a relative tolerance.